src/views/ViewPoint.js

```diff
@@ -1,19 +1,19 @@
 import React, { useCallback } from 'react';
 import { Just } from 'folktale/maybe';
-import { Input, H1, P } from 'indigo-react';
+import { Input, Grid, Text } from 'indigo-react';
 
 import { useHistory } from 'store/history';
 import { usePointCursor } from 'store/pointCursor';
 
-import { ROUTE_NAMES } from 'lib/routeNames';
 import patp2dec from 'lib/patp2dec';
 import { usePointInput } from 'lib/useInputs';
 
 import View from 'components/View';
 import { ForwardButton } from 'components/Buttons';
+import ViewHeader from 'components/ViewHeader';
 
 export default function ViewPoint() {
```

Comment:
This isn't routed to from anywhere anymore, I think we can rip this out? (Missed this during my teardown.)